.chronus/changes/arm-lro-opt-2024-5-5-2-57-50.md

```diff
@@ -0,0 +1,12 @@
+---
+# Change versionKind to one of: internal, fix, dependencies, feature, deprecation, breaking
+changeKind: feature
+packages:
+  - "@azure-tools/typespec-autorest"
+  - "@azure-tools/typespec-azure-core"
+  - "@azure-tools/typespec-azure-resource-manager"
+---
+
+- Add support for displaying lro options in typespec-autorest based on lro metadata
+- Add overrides for lro resolution when multiple resolution pathways exist
+- Remove OpenAPI dependencies form ARM LRO templates
```

Comment:
Ideally this should be creatred as individual changelog files that match their corresponding package, otherwise here all 3 lines will show up in all 3 packages changelog